src/main/java/pro/beam/api/resource/chat/methods/AuthenticateMessage.java

```diff
@@ -21,4 +22,12 @@ public AuthenticateMessage() {
     }
 
     public Object[] arguments;
+
+    /**
+     * Signals the chat servers to send a partial chat "history" as new, incoming messages
+     * to catch newly-connecting clients up on old messages.
+     */
+    public void sendHistory() {
+        this.arguments[3] = "TODO";
```

Comment:
@JamyDev not sure what to put here in order to get chat history. Any help would be appreciated!